src/core/dbus-execute.c

```diff
@@ -1826,6 +1862,33 @@ int bus_exec_context_set_transient_property(
         if (streq(name, "LogRateLimitBurst"))
                 return bus_set_transient_unsigned(u, name, &c->log_ratelimit_burst, message, flags, error);
 
+        if (streq(name, "LogFilterPatterns")) {
+                char *pattern;
+                bool is_allowlist;
```

Comment:
Please use `int` instead of `bool` when reading dbus message with type `b`.

Reply:
`sd-bus()` stores boolean value to `int`, to support old C, which does not support `bool`.
Unfortunately, we do not use `bool` for public functions.